### PR TITLE
gsoap: remove dead source url, bump to v2.8.134

### PR DESCRIPTION
**Pull request by saukijan**
Specify library name and version:  **gsoap/2.8.134**

Closes Issue #24287 and Issue #23454

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by perseoGI — APPROVED** [review]
Thanks! LGTM

**Review by RubenRBS — COMMENTED** [review]
Hi! Thanks a lot! I'll handle the mirroring on our own artifactory, no need for further action from you, but I have a question!

Why dud you remove the old versions? Those are still available under our mirror :)

**Comment by RubenRBS** [comment]
Sorry for the noise @saukijan

**Comment by saukijan** [comment]
Hi @RubenRBS, are you sure that they are available? I checked [c3i.jfrog.io](https://c3i.jfrog.io) yesterday and gsoap sources were not there. Also, as stated in Issue #23454, [gsoap_2.8.132 from c3i](https://c3i.jfrog.io/artifactory/cci-sources-backup/sources/gsoap/gsoap_2.8.132.zip) returns a 404 code, I haven't tested gsoap_2.8.129 from c3i thought...

**Comment by danimtb** [comment]
> Hi @RubenRBS, are you sure that they are available? I checked [c3i.jfrog.io](https://c3i.jfrog.io) yesterday and gsoap sources were not there. Also, as stated in Issue #23454, [gsoap_2.8.132 from c3i](https://c3i.jfrog.io/artifactory/cci-sources-backup/sources/gsoap/gsoap_2.8.132.zip) returns a 404 code, I haven't tested gsoap_2.8.129 from c3i thought...

Hi @saukijan, indeed the sources were missing there for gsoap 2.8.132. We have uploaded them from our sources backup repository, thanks for the notice. Please keep the old versions config.yml as well as the contributed new one. Thanks a lot and sorry for the noise 😄

**Comment by saukijan** [comment]
Thanks for the info @danimtb, I'll readd the original sources then

**Review by danimtb — DISMISSED** [review]

File: recipes/gsoap/all/conandata.yml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
   "2.8.132":
     url:
-      - "https://downloads.sourceforge.net/project/gsoap2/gsoap_2.8.132.zip"
       - "https://c3i.jfrog.io/artifactory/cci-sources-backup/sources/gsoap/gsoap_2.8.132.zip"
     sha256: "d6eb5d0d2c31532746f4dc9fa1ce95d4553414e918059eac23cf081d88c2aeee"
   "2.8.129":
     url:
-      - "https://downloads.sourceforge.net/project/gsoap2/gsoap_2.8.129.zip"
       - "https://c3i.jfrog.io/artifactory/cci-sources-backup/sources/gsoap/gsoap_2.8.129.zip"
     sha256: "16cb8852ea791a6aec8f0213d619c15eecc8171e0c888f3b0e0c66d3ef78e20a"
```
Comment (RubenRBS) [inline recipes/gsoap/all/conandata.yml:13]: Could you change this file to:

```
# The sources for this project get deleted from time to time.
# We use the backup sources from Conan Center as mirrors to let users compile in the future
# When adding new versions, directly add the URL following previous versions schema
# https://c3i.jfrog.io/artifactory/conan-center-backup-sources/ + sha256 of the sources
# The file will be automatically uploaded after the firs successful compilation of the new version
# And will be usable as a mirror from then on.
# At some point the Conan client might come pre-configured with this backup remote by default, but dont delete the links then
# some users might still rely on the old revisions/not have it set to point to our CCI backup remote 
sources:
  "2.8.134":
    url:
      - "https://sourceforge.net/projects/gsoap2/files/gsoap_2.8.134.zip/download"
      - "https://c3i.jfrog.io/artifactory/conan-center-backup-sources/63478e555c0ccde0164f055ff605b02805db0abc6712a04bcb14cb617b047218"
    sha256: "63478e555c0ccde0164f055ff605b02805db0abc6712a04bcb14cb617b047218"
  "2.8.132":
    url:
      - "https://c3i.jfrog.io/artifactory/conan-center-backup-sources/d6eb5d0d2c31532746f4dc9fa1ce95d4553414e918059eac23cf081d88c2aeee"
    sha256: "d6eb5d0d2c31532746f4dc9fa1ce95d4553414e918059eac23cf081d88c2aeee"
  "2.8.129":
    url:
      - "https://c3i.jfrog.io/artifactory/conan-center-backup-sources/16cb8852ea791a6aec8f0213d619c15eecc8171e0c888f3b0e0c66d3ef78e20a"
    sha256: "16cb8852ea791a6aec8f0213d619c15eecc8171e0c888f3b0e0c66d3ef78e20a"
```

I can't directly push to your fork :)

Comment (saukijan) [reply recipes/gsoap/all/conandata.yml]: All done, thanks for your help!

**Comment by saukijan** [comment]
Sorry for the multiple commits that do the same thing, github online editor was not cooperating and insisted on adding trailing whitespaces at the end of the file

**Comment by RubenRBS** [comment]
This is an issue on our end, fixing!

**Comment by RubenRBS** [comment]
Hi @saukijan could you please merge from master and update this branch? We have now fixed the issue keeping this PR from being able to get merged :)

**Review by perseoGI — APPROVED** [review]

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**